pipeline/matching/generators/similar_intents.py

```diff
@@ -1,7 +1,17 @@
 from typing import Iterator
 
+import pytest
+
 from pipeline.matching.core import MatchGenerator
-from pipeline.types import Match, MatchingInput
+from pipeline.types import (
+    Intent,
+    IntentMatch,
+    Match,
+    MatchingInput,
+    MatchMetadata,
+    Side,
+    User,
+)
 
 GENERATOR_SIMILAR_INTENTS = "similarIntentsGenerator"
 
```

Comment:
Maybe we should change the default value of `min_common: int = 0` to `1`, otherwise we will generate matches for users with no compatible intents.

Reply:
Got it. thanks!

Reply:
> Maybe we should change the default value of `min_common: int = 0` to `1`, otherwise we will generate matches for users with no compatible intents.
Resolved on line 24